src/rules/no-restricted-paths.js

```diff
@@ -29,8 +29,8 @@ module.exports = {
             items: {
               type: 'object',
               properties: {
-                target: { type: 'string' },
-                from: { type: 'string' },
+                target: { type: ['string', 'array'] },
+                from: { type: ['string', 'array'] },
```

Comment:
i think this needs to be `oneOf`, so it can be either a string, or an array of unique strings

Reply:
@ljharb agree! fixed in https://github.com/import-js/eslint-plugin-import/pull/2466/commits/bd987a55ff959f3239cee4f7506ef5f30ac96448